defi/src/api2/routes/index.ts

```diff
@@ -113,6 +113,9 @@ export default function setRoutes(router: HyperExpress.Router, routerBasePath: s
   router.get("/overview/:type", ew(getOverviewFileRoute))
   router.get("/overview/:type/:chain", ew(getOverviewFileRoute))
   router.get("/summary/:type/:name", ew(getDimensionProtocolFileRoute))
+  router.get("/summaryBreakdown/:type/:name", ew(getDimensionBreakdownProtocolFileRoute))
```

Comment:
what I would do here is, instead of adding these under normal router, will create a new pro api router and add it there